oblique-breadcrumb/src/app/app.module.ts

```diff
@@ -47,4 +48,9 @@ registerLocaleData(localeITCH);
   ],
   bootstrap: [AppComponent],
 })
-export class AppModule {}
+export class AppModule {
+  constructor() {
+    const masterLayoutConfig = inject(ObMasterLayoutConfig)
+    masterLayoutConfig.homePageRoute = "/breadcrumb";
```

Comment:
following the pattern used on other pages, i believe this should be `/oblique-breadcrumb`